css/index.css

```diff
@@ -1 +1,477 @@
+}
+
+.top-content .button {
+	margin-bottom: 1.7%;
+	font-size: 1.7rem;
+	text-align: center;
+	border: 2px solid black;
+	padding: 5px 16.7%;
+	margin-top: .6%;
+	background-color: white;
+	cursor: pointer;
+}
+
+.button:hover {
+	background-color: pink;
+}
+
+.button:active {
+	background-color: black;
+	color: white;
+	box-shadow: 0 6px #FFA500;
+	transform: transformY(4px);
+}
+
+.middleContent h3 {
+	margin-top: 6%;
+	margin-left: 1.7%;
+	font-size: 2rem;
+	display: flex;
+	justify-content: flex-start;
+}
+
+.middleContent {
+	display: flex;
+	justify-content: center;
+	flex-wrap: wrap;
+	font-size: 1.6rem;
+	border-top: 2px solid black;
+	border-bottom: 2px solid black;
+}
+
+.topMiddle-text {
+	display: flex;
+	margin-bottom: 2.9%;
+}
+
+.sectionText {
+	display: flex;
+	text-align: left;
+	padding-left: 1.7%;
+}
+
+.features {
+	flex: 1;
+}
+
+.about {
+	flex: 1;
+}
+
+.bottom-text {
+	display: flex;
+	margin-top: 1.1%;
+	margin-bottom: 1.7%;
+	padding-bottom: 1.7%;
+}
+
+.services {
+	flex: 1;
+}
+
+.product {
+	flex: 1;
+}
+
+.vision {
+	flex: 1;
+}
+
+.ftr-info {
+	display: flex;
+	font-size: 1.6rem;
+	flex-direction: column;
+}
+
+.ftr-info h3 {
+	margin-top: 6%;
+	margin-left: 1.7%;
+	display: flex;
+	justify-content: flex-start;
+}
+
+.ftr-text {
+	padding-left: 1.7%;
+}
+
+.copyright-bottom {
+	text-align: center;
+	font-size: 1.6rem;
+	margin-bottom: 11.4%;
+}
+
+/* below css is service page specific */
+
+.services-header-img {
+	margin: 30px 0%;
+}
+
+.service-top h2 {
+	font-size: 2.5rem;
+}
+
+.service-top {
+	padding-bottom: 2.9%;
+	font-size: 1.6rem;
+	border-bottom: 1px solid black;
+}
+
+.middle-container {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-between;
+	padding-top: 3.4%;
+	border-bottom: 1px solid black;
+}
+
+.middle-blocks {
+	border: 1px solid black;
+	margin-bottom: 4%;
+	font-size: 1.6rem;
+	width: 48.3%;
+	background-color: whitesmoke;
+	padding: 2.3%;
+}
+
+.middle-blocks button {
+	border: 1px solid black;
+	padding: 6px 4.5%;
+	margin-top: 2.3%;
+	background-color: white;
+	border-radius: 10%;
+}
+
+.footer-container {
+	padding: 30px 0%;
+	display: flex;
+}
+
+.service-info-img {
+	margin-bottom: 2.8%;
+}
+
+.bottom-fact-block {
+	padding-right: 3.4%;
+	font-size: 1.6rem;
+}
+
+.bottom-fact-block ul {
+	margin-top: 2.8%;
+}
+
+.bottom-fact-block ul li {
+	margin-top: 1.7%;
+	font-size: 1.6rem;
+	list-style-type: square;
+	margin-left: 2.3%;
+}
+
+@media (max-width:800px) {
+	.container {
+		width: 100%;
+	}
+	.container header {
+		flex-direction: column-reverse;
+		align-items: center;
+		width: 100%;
+	}
+	.container .navbar {
+		width: 100%;
+		text-align: center;
+	}
+	.container .logo {
+		display: block;
+	}
+	.navbar .topNav {
+		margin: 0px 1.9%;
+	}
+	.top-img {
+		display: none;
+	}
+	.top-content {
+		height: auto;
+		width: 100%;
+		justify-content: center;
+		margin: 45px 0%;
+	}
+	.top-content .button {
+		margin-bottom: .6%;
+		padding: 7px 14%;
+		font-size: 17px;
```

Comment:
Great job converting most of the font-sizes! 👍 This is the only one I found that still used pixels.